### DISCUSSION TITLE
Running HWiNFO64 causes audio offset in osu! lazer

### DISCUSSION THREAD

**Discussion by Packsolite** [ppy/osu#29613] [Q&A]
### Type

Game behaviour

### Bug description

I have no idea why but when i have HWiNFO64 running and play osu, there is a weird lag at the start of every beatmap that causes the audio track to be offset. It gets worse and worse the more i play.

Software confirmed causing issues:
- HWiNFO64
- Voicemeeter

### Screenshots or videos

Notice the short freeze at the beginning of each beatmap.
https://youtu.be/v1H59-SJbeQ
https://youtu.be/IjwatJqvomc

### Version

2024.625.2-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16241930/compressed-logs.zip)

**Comment by smoogipoo** [comment]
Very weird. You say it's with that program running though I can repro to a much smaller/mostly unnoticeable degree without it.

**Comment by Packsolite** [comment]
> Very weird. You say it's with that program running though I can repro to a much smaller/mostly unnoticeable degree without it.

Yeah, the lag is always there, but it's barely noticable usually. But with the monitoring software running, the game is literally unplayable.

**Comment by PasswordOnOsu** [comment]
I had the same issue. What solved it for me was turning off the audio monitoring on voicemeeter for every option. For some reason it interfered with the syncing.

**Comment by PasswordOnOsu** [comment]
> I had the same issue. What solved it for me was turning off the audio monitoring on voicemeeter for every option. For some reason it interfered with the syncing.

it seems like being in a discord call could stop this solution from working? otherwise it works fine when im not in a call from what i can tell.

**Comment by Vexcited** [comment]
Happens to me without having any third party software running in background, not sure if there's any hidden background system service that might cause this though. Played osu!lazer on Windows and Linux (Arch) and both OSes caused this audio offset...

**Comment by Packsolite** [comment]
Seems like BTMC has the same problem in his streams: https://youtu.be/Snn9ZNt9WTQ?t=375

**Comment by peppy** [comment]
It's not the same problem. BTMC found out he had a wack global offset.

**Comment by Packsolite** [comment]
> It's not the same problem. BTMC found out he had a wack global offset.

You sure about that? He later mentioned that he closed some application "real.exe" and it fixed it. It seems like he put the "wack global offset" because of that issue...

**Comment by peppy** [comment]
> You sure about that? He later mentioned that he closed some application "real.exe" and it fixed it. It seems like he put the "wack global offset" because of that issue...

even more sure now. yes

**Comment by peppy** [comment]
Are there any improvements to this in the latest release? We updated some audio libraries which may have made this better or worse.

**Comment by Packsolite** [comment]
> Are there any improvements to this in the latest release? We updated some audio libraries which may have made this better or worse.

So i just tried and bad news. Might be placebo, but i think it's actually worse now. 

Also on top of the weird audio delay, i just had something really strange happen. Approach circles look like they render in like 15fps.
Heres a 60fps video. On an 144hz monitor the issue is even more noticeable.
https://youtu.be/boAWo8_D2sw
No, the video is not lagging, it's actually looking like that ingame.
If you go frame by frame you can see the approach circle animation is not smooth but very jumpy, sometimes barely mooving from one frame to the next and then jumping.

**Comment by peppy** [comment]
At that point you just need to close the software. It's in no way our fault.
